indexing-service/src/main/java/org/apache/druid/indexing/common/actions/SegmentAllocationQueue.java

```diff
@@ -364,7 +364,11 @@ private boolean processBatch(AllocateRequestBatch requestBatch)
     final Set<DataSegment> updatedUsedSegments = retrieveUsedSegments(requestKey);
 
     if (updatedUsedSegments.equals(usedSegments)) {
-      requestBatch.failPendingRequests("Allocation failed probably due to conflicting segments.");
+      log.info(
```

Comment:
There should be some stuff interpolated into this log message to make it more useful to an operator, such as:

- name of datasource
- name of task that tried to do the allocation (if available)
- interval for the allocation (if available)

Reply:
Updated to include the batch key, which has datasource, groupId and preferredAllocInterval.